pkg/handlers/search/delete_by_query_test.go

```diff
@@ -0,0 +1,159 @@
+package search
+
+import (
+	"github.com/goccy/go-json"
+	"github.com/stretchr/testify/assert"
+	"github.com/zinclabs/zinc/pkg/core"
+	"github.com/zinclabs/zinc/pkg/ider"
+	"github.com/zinclabs/zinc/pkg/meta"
+	"github.com/zinclabs/zinc/test/utils"
+	"net/http/httptest"
+	"testing"
+	"time"
```

Comment:
the import section, should split to three part:
1. standard library
2. third part library
3. own part library

you can use vscode install go plugin to format it.

like above, we should format to:

```
"net/http/httptest"
"testing"
"time"

"github.com/goccy/go-json"
"github.com/stretchr/testify/assert"

"github.com/zinclabs/zinc/pkg/core"
"github.com/zinclabs/zinc/pkg/ider"
"github.com/zinclabs/zinc/pkg/meta"
"github.com/zinclabs/zinc/test/utils"
```